app/vminsert/common/streamaggr.go

```diff
@@ -33,7 +33,7 @@ var (
 )
 
 var (
-	saCfgReloaderStopCh chan struct{}
+	saCfgReloaderStopCh = make(chan struct{})
```

Comment:
nit: should this be present in this PR?

Reply:
it should, see https://github.com/VictoriaMetrics/VictoriaMetrics/commit/da60a68d09b7cf4eb371783062b30a22d24656b1#diff-90e4ac1ebceb440f18ef46a8c518b96bccf186298fff74e68ee00972e02e7348R62